packages/browser/src/core/stats/remote-metrics.ts

```diff
@@ -85,7 +89,7 @@ export class RemoteMetrics {
     }
 
     this.queue.push({
-      type: 'Counter',
+      type: 'counter',
```

Comment:
I think this change might break some of our metrics.

Reply:
oh yeah, I just thought of that =S I was going to check if our API required title case or not.